docs/manual/cli/maintenance-mode.de.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "contao:maintenance-mode"
+description: "Back- und Frontend in den Wartungsmodus versetzten."
+aliases:
+    - /de/cli/maintenance-mode/
+---
+
+
+{{< version "4.13" >}}
+
+Der Wartungsmodus wurde für diese Contao-Version neugeschrieben. Über die Kommandozeile kann die komplette Contao-Installation 
+(Back- und Frontend) in den Wartungsmodus versetzt werden. Diese Funktion ist sehr nützlich, wenn du eine 
+Aktualisierung an deinem Systems vornehmen möchtest.
+
+Ausserdem hast du die Möglichkeit das Frontend für jeden 
+[Startpunkt einer Website](../../layout/seitenstruktur/seiten-konfigurieren/#website-einstellungen) in den 
+Wartungsmodus zu versetzen.
+
+
+```bash
+php vendor/bin/contao-console contao:maintenance-mode [options] [<state>]
+```
+
+| Option                          | Beschreibung                                                                                                                                                        |
+|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--template=TEMPLATE`           | Ermöglicht die Verwendung eines anderen Twig-Templates-Namens, wenn der Wartungsmodus aktiviert wird. Standard ist `@ContaoCore/Error/service_unavailable.html.twig` |
+| `--templateVars[=TEMPLATEVARS]` | Füge dem Twig-Template benutzerdefinierte Template-Variablen hinzu, wenn der Wartungsmodus aktiviert wird (als JSON bereitstellen). Standard ist `{}`               |
+| `--format=FORMAT`               | Du kannst zwichen den Ausgabeformaten `txt` und `json` wählen. Standard ist `txt`.                                                                                  |
```

Comment:
```suggestion
| `--format=FORMAT`               | Du kannst zwichen den Ausgabeformaten `txt` und `json` wählen. Standard ist `txt`.                                                                                  |
```

This option exists for all commands that the Contao Manager also needs to use and is mostly important for the Contao Manager (or for advanced usage otherwise). imho we do not need to document that in the manual.